packages/react/src/useTable.tsx

```diff
@@ -205,8 +228,25 @@ export const useTable = <
         },
         sortable: "sortable" in field && field.sortable,
         relatedField,
-      } as TableColumn;
+        isCustomCell: false,
+      } satisfies RecordTableColumnValue;
     });
+
+    if (options?.columns) {
```

Comment:
Isn't this always going to add the custom columns at the end of the column list? I think they should be in the order they're passed.

Reply:
Updated to sort the column result based on the columns option